VideoLocker/res/layout/fragment_handout.xml

```diff
@@ -2,8 +2,12 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/grey_act_background"
-    android:orientation="vertical" >
+    android:background="@color/edx_grayscale_neutral_white"
+    android:orientation="vertical"
+                android:paddingLeft="@dimen/common_course_display_margin_side"
+                android:paddingRight="@dimen/common_course_display_margin_side"
+                android:paddingTop="@dimen/common_course_display_margin_top"
+                android:paddingBottom="@dimen/common_course_display_margin_top">
```

Comment:
Also, could you remove this padding and at it to the webview? Otherwise the scroll indicators are inset, looks wrong.

![padding](https://cloud.githubusercontent.com/assets/1685613/12273912/57bd567c-b936-11e5-8f26-de5f6291520e.png)